web/src/scenes/app/index.tsx

```diff
@@ -20,12 +20,20 @@ export const App = memo(function App(props: { appId: string }) {
   return (
     <AuthRequired>
       <Layout>
-        {(isLoading || !currentApp) && (
+        {isLoading && (
           <div className="w-full h-full flex justify-center items-center">
             <Preloader className="w-20 h-20" />
           </div>
         )}
 
+        {!isLoading && !currentApp && (
+          <div className="grid">
```

Comment:
@michaelfurutin Let's make this centered.

Reply:
done